src/recall.cpp

```diff
@@ -126,7 +126,9 @@ static int roff_print_line;       // Place to print line now being loaded.
 // Number of kills needed for information.
 
 // the higher the level of the monster, the fewer the kills you need
-#define knowarmor(l, d) ((d) > 304 / (4 + (l)))
+unsigned int knowarmor(unsigned int l, unsigned int d) {
+	return d > 304u / (4u + l);
+}
```

Comment:
As this macro is only being called in one place, how about we just inline it and delete the macro completely?

The calling function (perhaps also including a comment) would be something like:

```
static void memoryKillDifficulty(Creature_t *creature, uint32_t monster_kills) {
    // Original knowarmor macro inlined
    if (monster_kills <= 304 / (4 + creature->level)) {
        return;
    }

    // ...
}
```